saltstack/final/salt/cleanup/user.sls

```diff
@@ -0,0 +1,7 @@
+{% if grains['os'] == 'CentOS' %}
+
+centos:
```

Comment:
similarly to the other pr, I prefer states with names to avoid state name collision. please give a proper name to the state

Reply:
Fixed, thanks